src/main/com/algos/src/maths/AbsoluteMax.java

```diff
@@ -13,12 +13,12 @@ public static int getMaxValue(int... numbers) {
         if (numbers == null || numbers.length == 0) {
             throw new IllegalArgumentException("Numbers array cannot be empty or null");
         }
-        int absMax = numbers[0];
+        int absMax = 0;  
```

Comment:
**Issue**: Initializing 'absMax' to 0 can lead to incorrect behavior if all numbers are negative, as it would incorrectly consider 0 as the maximum absolute value. <br> **Fix**: Initialize 'absMax' with the absolute value of the first element of the array to correctly handle arrays with all negative numbers. <br> **Code Suggestion**: 
 ```
 -        int absMax = 0;
 +        int absMax = Math.abs(numbers[0]);
 ```